src/cairoWriter.ts

```diff
@@ -206,10 +207,9 @@ class VariableDeclarationStatementWriter extends CairoASTNodeWriter {
 
     const documentation = getDocumentation(node.documentation, writer);
     const declarations = node.assignments.flatMap((id) => {
-      if (id === null)
-        throw new NotSupportedYetError(
-          `VariableDeclarationStatements with gaps not implemented yet`,
-        );
+      if (id === null) {
+        return [`__warp_gv${this.gapVarCounter++}`];
+      }
```

Comment:
Out of interest, where would there be an empty variable declaration?

Reply:
Usually something like `(uint a,) = f()` where the person doesn't care about the second return from f

Reply:
I thought these would have been handled by the tuple filler? or was that only for assignments and not vardecl statements?

Reply:
Yep, they aren't actually TupleExpressions, they're just VariableDeclarationStatements with more slots than children

Reply:
HA ok cool.